nomic/dataset.py

```diff
@@ -671,7 +700,7 @@ def _download_large_feather(self, dest: Optional[Union[str, Path]] = None, overw
             elif sidecars is None:
                 sidecars = set()
             if not "." in rawquad:
-                for sidecar in sidecars:
+                for sidecar in sidecars | registered_sidecars:
```

Comment:
is | and `or` one and the same. Also a question

Reply:
unrelated, these changes were rebased in not sure why github shows the diff.

Reply:
> is | and `or` one and the same. Also a question

even though its unrelated, they are not. `or` is equivalent to `||`, not `|`.  `|` is either bitwise-or (for numbers) or in this case, union of sets